src/app/manager-dashboard/reports/reports-detail.component.ts

```diff
@@ -236,6 +239,17 @@ export class ReportsDetailComponent implements OnInit, OnDestroy {
     }
   }
 
+  getTeams() {
+    this.couchService.findAll('teams', { 'selector': { 'status': 'active' } }).subscribe((teams: any) => {
+      this.teams = teams.filter((team: any) => team.teamPlanetCode === this.planetCode);
+    });
+  }
+
+  onTeamsFilterChange(filterValue) {
+    this.couchService.findAll('teams', findDocuments({ teamId: filterValue._id, docType: 'membership' }))
+      .subscribe((teams: any) => this.filter.teams = teams);

```

Comment:
You also need to call `filterData` to trigger the data change based on the filter.